onnxruntime/core/providers/cpu/nn/conv_transpose_attributes.h

```diff
@@ -212,7 +212,7 @@ struct ConvTransposeAttributes : public ConvAttributes {
     if (pad_type == AutoPadType::SAME_UPPER || pad_type == AutoPadType::SAME_LOWER) {
       // total pad
       auto total_pad = ComputeTotalPad(in_size, stride, adj,
-                                       kernel, dilation, in_size);
+                                       kernel, dilation, in_size * stride);
```

Comment:
Do all 4 models needing this give results aligning with tf after this ?

Reply:
Two models are working good with this change. The other two models have other issues which are unrelated.